imblearn/over_sampling/random_over_sampler.py

```diff
@@ -23,29 +23,54 @@ class RandomOverSampler(BaseOverSampler):
 
     Parameters
     ----------
-    ratio : str, dict, or callable, optional (default='auto')
-        Ratio to use for resampling the data set.
-
-        - If ``str``, has to be one of: (i) ``'minority'``: resample the
-          minority class; (ii) ``'majority'``: resample the majority class,
-          (iii) ``'not minority'``: resample all classes apart of the minority
-          class, (iv) ``'all'``: resample all classes, and (v) ``'auto'``:
-          correspond to ``'all'`` with for over-sampling methods and ``'not
-          minority'`` for under-sampling methods. The classes targeted will be
-          over-sampled or under-sampled to achieve an equal number of sample
-          with the majority or minority class.
-        - If ``dict``, the keys correspond to the targeted classes. The values
-          correspond to the desired number of samples.
-        - If callable, function taking ``y`` and returns a ``dict``. The keys
+    sampling_target : float, str, dict or callable, (default='auto')
+        Sampling information to resample the data set.
+
+        - When ``float``, it correspond to the ratio :math:`\\alpha_{os}`
+          defined by :math:`N_{rm} = \\alpha_{os} \\times N_{m}` where
+          :math:`N_{rm}` and :math:`N_{M}` are the number of samples in the
+          minority class after resampling and the number of samples in the
+          majority class, respectively.
+
+        .. warning::
```

Comment:
if you indent this two spaced, then it is included in the list (which is better I think)